scripts/deploy-mainnet-compound-usdt.ts

```diff
@@ -0,0 +1,79 @@
+import 'tsconfig-paths/register';
+
+import { deployBondModel, deployCompoundController, deployCompoundProvider, deployJuniorBond, deploySeniorBond, deploySmartYield, deployYieldOracle } from '@testhelp/index';
+import { Wallet, BigNumber as BN } from 'ethers';
+import { run, ethers } from 'hardhat';
+import { ERC20Factory } from '@typechain/ERC20Factory';
+
+const A_HOUR = 60 * 60;
+const A_DAY = A_HOUR * 24;
+
+const seniorBondCONF = { name: 'BarnBridge cUSDT sBOND', symbol: 'bb_sBOND_cUSDT' };
+const juniorBondCONF = { name: 'BarnBridge cUSDT jBOND', symbol: 'bb_jBOND_cUSDT' };
+const juniorTokenCONF = { name: 'BarnBridge junior cUSDT', symbol: 'bb_cUSDT' };
+
+const oracleCONF = { windowSize: 3 * A_DAY, granularity: 4 };
+
+// barnbridge
+const decimals = 8; // same as USDT
+const feesOwner = '0x4cAE362D7F227e3d306f70ce4878E245563F3069';
+
+// externals ---
+
+// compound
+const cUSDT = '0xf650c3d88d12db855b8bf7d11be6c55a4e07dcc9';
+const COMP = '0xc00e94cb662c3520282e6f5717214004a7f26888';
+
+const USDT = '0xdac17f958d2ee523a2206206994597c13d831ec7';
+const WETH = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2';
+const uniswapPath = [COMP, WETH, USDT];
+
+async function main() {
+
+  return;
+
+  const [deployerSign, ...signers] = (await ethers.getSigners()) as unknown[] as Wallet[];
+
+  console.log('Deployer:', deployerSign.address);
+  console.log('Others:', signers.map(a => a.address));
+
+  const bondModel = await deployBondModel(deployerSign);
```

Comment:
for compound can you please use the `BondModelV2Compounded` bond model as that is the latest one used by the other live compound smartYields, with a 65% MAX_POOL_RATION? https://etherscan.io/address/0x48a41e77bef0654831c12cd816bd62ce9c5adbb8#readContract